pyproject.toml

```diff
@@ -97,9 +97,11 @@ murfey = "murfey.client:run"
 "password" = "murfey.server.api.auth:password_token_validation"
 [project.entry-points."murfey.config.extraction"]
 "murfey_machine" = "murfey.util.config:get_extended_machine_config"
-[project.entry-points."murfey.workflows"]
-"lif_to_stack" = "murfey.workflows.lif_to_stack:zocalo_cluster_request"
-"tiff_to_stack" = "murfey.workflows.tiff_to_stack:zocalo_cluster_request"
+[project.entry-points."murfey.workflows.clem"]
```

Comment:
I wonder if we want the ".clem" bit in the name. We probably do for now so it's clear what is happening, but it might be more general to remove that when we migrate other parts of the code?